Assignment4.java

```diff
@@ -0,0 +1,129 @@
+
+package assignment4;
+
+import java.util.Scanner;
+
+public class Assignment4 {
+
+    /**
+     * Curtis Kunkle
+     * Assignment 4
+     * This program uses methods to display the results
+     * of a multiple choice survey.
+     */
+    public static void main(String[] args) {
+        
+        //declare character for user input
+        int choice;
+        
+        //new scanner object!
+        Scanner keyboard = new Scanner(System.in);
+        
+        //Output prompt, list killer Peavey tube amps
+        print("What is the best high-gain Peavey tube Amplifier?");
+        print("1: Classic Series");
+        print("2: ValveKing");
+        print("3: 6505");
+        print("4: XXX");
+        print("5: JSX");
+        print("6: Butcher");
+        print("7: QUIT");
+        
+        //accepts the first integer of the user's input
+        choice = keyboard.nextInt();
+        
+        //error checking.  
+        while(choice != 1 && choice != 2 &&
+              choice != 3 && choice != 4 &&
+              choice != 5 && choice != 6 && choice != 7){
+         print("Invalid choice! Try again.");
+         choice = keyboard.nextInt();
+        }
```

Comment:
The while could start to get a bit long and hard to manage when listing every available choice. Perhaps using <0 and >7 would be better. Then if it gets longer, the only change that would be needed would be to the upper value. (7).